README.md

```diff
@@ -71,7 +71,9 @@ public function registerBundles()
 
 ### Configuration example
 
-You can configure default query parameter names and templates
+You can configure default query parameter names and templates.
+
+You need to create a file "config/packages/knp_paginator.yaml". The file extension must be "yaml" and not "yml".
```

Comment:
Both these sentences are based on Symfony conventions, and so not necessarily true.
Indeed, you can have a Symfony 6 project in which your configuration is stored elsewhere, even in a `.yml` file.

Reply:
My project is actually on Symfony 6.3 and when I put "knp_pagination" in ".yml", it does not works...

Reply:
It doesn't work because the default Symfony kernel is looking for .yaml files, but that's not a requirement. It's just a default.